File: programmers/level1/문자열_다루기_기본.java
package programmers;

public class 문자열_다루기_기본 {
	static
	class Solution {
		  public boolean solution(String s) {
		      if(s.length()==4 || s.length() == 6) {
		    	  for(char c : s.toCharArray()) {
		    		  if(c > '9' || c < '0') {
		    			  return false;
		    		  }
		    	  }
		    	  return true;
		      }
		      return false;
		  }
		}
	public static void main(String[] args) {
		String s = "1234";
		Solution sol = new Solution();
		System.out.println(sol.solution(s));
	}
}

File: programmers/js/new/level1/문자열_다루기_기본.js
function solution(s) {
    if (s.length !== 4 && s.length !== 6) {
        return false;
    }
    for (let i = 0; i < s.length; ++i) {
        if (!(s.charAt(i) >= "0" && s.charAt(i) <= "9")) {
            return false;
        }
    }
    return true;
}
